### PR TITLE
Adding Tinker Board detection

### PR DESCRIPTION
**Pull request by BlitzCityDIY**
Adding detection for the ASUS Tinker Board SBC.
SoC: Rockchip 3288 (RK3288)

### CODE REVIEW

**Comment by makermelissa** [comment]
Thanks, 2 things you may want to look at: 

1. You will want to make sure it's being detect in the `any_embedded_linux` function.
2. There's 2 tinker board variations, as I'm sure you're aware (Tinker Board and Tinker Board S), so maybe structure it so it's easy to add the second one such as creating an any_tinker_board function in the board.py file.

**Comment by BlitzCityDIY** [comment]
okay, thanks! will update.

**Comment by makermelissa** [comment]
It looks like it just needs to be Black formatted. You should be able to install it on your computer with: `pip3 install black`
Then just run `black --target-version=py35 .` and it will automatically format your files correctly.

**Comment by BlitzCityDIY** [comment]
thanks for the info on Black formatting! i'm seeing that pylint error- going to work on it.

File: adafruit_platformdetect/board.py [hunk 1]
```diff
@@ -119,6 +119,8 @@ def id(self):
             board_id = self._clockwork_pi_id()
         elif chip_id == chips.RK3308:
             board_id = self._rock_pi_id()
+        elif chip_id == chips.RK3288:
+            board_id = boards.ASUS_TINKER_BOARD
```
Comment (makermelissa) [inline adafruit_platformdetect/board.py:123]: Did you mean to use `_asus_tinker_board_id` here?

Comment (BlitzCityDIY) [reply adafruit_platformdetect/board.py:123]: i was having trouble with that earlier trying to get it to pass lint but i think it was more _asus_tinker_board_id function returning something else. it's passing now though so will commit again with that update shortly.

Comment (makermelissa) [reply adafruit_platformdetect/board.py:123]: Ok, sounds good. I just wanted to make sure.

File: adafruit_platformdetect/board.py [hunk 2]
```diff
@@ -409,6 +419,10 @@ def any_udoo_board(self):
         """Check to see if the current board is an UDOO board"""
         return self.id in boards._UDOO_BOARD_IDS
 
+    def any_asus_tinker_board(self):
+        """Check to see if the current board is an ASUS Tinker Board"""
+        return self.ASUS_Tinker_Board
```
Comment (makermelissa) [inline adafruit_platformdetect/board.py:424]: This function should return a boolean value, so maybe change it to `return self.id == self.ASUS_Tinker_Board`

Comment (BlitzCityDIY) [reply adafruit_platformdetect/board.py:424]: will do and will test

**Comment by BlitzCityDIY** [comment]
i had to merge to update branches after i had made changes locally- it looks okay but if anything is funky that is probably why and i'll redo. again, i really appreciate your patience and guidance on this!

File: adafruit_platformdetect/board.py (hunk 1 above)
Comment (makermelissa) [reply adafruit_platformdetect/board.py:123]: Hi, just curious if you were planning on making this change in this pull request?

**Comment by makermelissa** [comment]
Hi @BlitzCityDIY, were you planning on updating the code to use `_asus_tinker_board_id()`? I know you were working on that guitar tuner project, so I understand if this got pushed aside for that. I'm just wondering how we should proceed here. Thanks.

**Comment by BlitzCityDIY** [comment]
Hi @makermelissa ! I have to admit I got a little lost here. I'll work on this this afternoon

**Comment by makermelissa** [comment]
Thanks. Let me know if you have any questions.

**Comment by BlitzCityDIY** [comment]
Hi @makermelissa ! when I submitted this it didn't seem to go through the usual checks/processes. Should I try committing again or is it okay?

**Comment by makermelissa** [comment]
It's ok. The reason I haven't merged it yet is because I thought you were still working on it since you haven't made use of the _asus_tinker_board_id() function.

**Comment by BlitzCityDIY** [comment]
thank you @makermelissa !! I learned a lot doing this.

**Comment by makermelissa** [comment]
You're welcome and thank you for adding this.

**Pull request merged**